modules/core/zio/src/weaver/ziocompat/Live.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package ziocompat
+
+import zio.{ IO, URLayer, ZEnv, ZIO }
+
+/**
+ * Service used for getting the real environment during tests. This is useful
+ * for timing or getting random values during tests. For example, getting a
+ * random port for starting servers or timing an action with the real clock.
+ * This pattern is inspired by ZIO-test
+ */
+object Live {
```

Comment:
This seems like a pretty overloaded term in zio-land. Any alternative name you can think of ?

Reply:
You are right that Live is an overloaded term, but AFAIK there is only one Live type/service in the zio libraries, and that is in zio-test for test purposes. It is familiar to zio-test users, who represent the majority of zio users. Only alternatives I can think of are `UnderlyingEnvironment` or `RealZEnv` or something similar. I am not sure these are better, but I'll carry on thinking

Reply:
You are right that Live is an overloaded term, but AFAIK there is only one Live type/service in the zio libraries, and that is in zio-test for test purposes. It is familiar to zio-test users, who represent the majority of zio users. Only alternatives I can think of are `UnderlyingEnvironment` or `RealZEnv` or something similar. I am not sure these are better, but I'll carry on thinking

Reply:
Cool, I'll trust your educated opinion on this :)